js-task-second/js/main.js

```diff
@@ -0,0 +1,82 @@
+const loader = document.getElementById('loader');
+const block = document.getElementById('block');
+const title = document.getElementById('title');
+const listOfArticles = [];
+
+async function requestOfDate () {
+  const response = await fetch('https://api.punkapi.com/v2/beers/random', {
+    method: "GET",
+    mode: "cors",
+    cache: "no-cache",
+    credentials: "same-origin",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    redirect: "follow",
+    referrerPolicy: "no-referrer",
+  });
+  return response.json();
+}
+
+function onClick (event) {
+  const target = event.currentTarget;
+  const lastElement = target.children.length - 1;
+  if (target.children[lastElement].style.display === 'flex') {
+    target.children[lastElement].style.display = 'none';
```

Comment:
once we have css file - I think it will be better to operate with the classnames, not styles